src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -205,10 +205,38 @@ public static bool MonitoringIsEnabled
         public event ResolveEventHandler ResourceResolve { add { } remove { } }
         //public void SetPrincipalPolicy(System.Security.Principal.PrincipalPolicy policy) { }
         /*
-        public void SetThreadPrincipal(System.Security.Principal.IPrincipal principal) 
-        { 
+        public void SetThreadPrincipal(System.Security.Principal.IPrincipal principal)
+        {
             if (principal == null) throw new ArgumentNullException("principal");
-            throw new PlatformNotSupportedException(); 
+            throw new PlatformNotSupportedException();
```

Comment:
The whitespace change is ok.  I'm just wondering why it's commented out (I see it was already that way). :smile:  Is there an issue associated with this getting uncommented?